src/Facebook/GraphNodes/GraphAlbum.php

```diff
@@ -110,9 +100,9 @@ public function getDescription()
     }
 
     /**
-     * Returns profile that created the album.
+     * Returns the profile that created the album.
      *
-     * @return GraphUser|null
+     * @return GraphNode|null
```

Comment:
This should still be `GraphUser`: https://developers.facebook.com/docs/graph-api/reference/v2.5/album